jsonrpc-specs/src/methods/history.json

```diff
@@ -0,0 +1,441 @@
+[
+  {
+    "name": "portal_historyRoutingTableInfo",
```

Comment:
I personally tend to like `portal_history_routingTableInfo` and `portal_state_routingTableInfo` a bit more.